resources/user-guides/index.md

```diff
@@ -67,7 +67,7 @@ title: "User Guides"
                 </div>
 <div class="row start-xs text-center" markdown="1">
 
-User Guides for Mining coming soon.
+[How to mine on a pool](mine-to-pool.html)
```

Comment:
Yeah I agree it should be changed. The procedure is almost the same for
the xmr-stak GPU miners, so should I just change the title to "How to
mine on a pool with xmr-stak-*" or similar?